denops_std/function/input.ts

```diff
@@ -86,7 +93,7 @@ export function inputlist(denops: Denops, textlist: string[]): Promise<number> {
  * Restore typeahead that was saved with a previous |inputsave()|.
  * Should be called the same number of times inputsave() is
  * called.  Calling it more often is harmless though.
- * Returns 1 when there is nothing to restore, 0 otherwise.
+ * Returns TRUE when there is nothing to restore, FALSE otherwise.
  */
 export function inputrestore(denops: Denops): Promise<void> {
   return denops.call("inputrestore") as Promise<void>;
```

Comment:
It's out of the scope of change, but I want to fix it.
```suggestion
export async function inputrestore(denops: Denops): Promise<boolean> {
  const result = await denops.call("inputrestore") as number;
  return !!result;
```

Reply:
Fixed. And also fixed `inputsave()`